### DISCUSSION TITLE
Welcome to tixi Discussions!

### DISCUSSION THREAD

**Discussion by joergbrech** [DLR-SC/tixi#221] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by namanshetty29** [comment]
> Hi, I am having trouble with the installation of TIXI, I am constantly getting errors in the initialization of 
![image](https://github.com/user-attachments/assets/50df90ab-9bbd-4004-b967-13f5bfc3343f)

tixi, I am not sure if its the wrong way to do or something with the installation steps( I am new into world with coding related stuff It would be great if you could help me with )

**Reply by joergbrech** [reply]
The error message is telling you, that the module tixi3 has no attribute Tixi, which is correct. Therefore the code `tixi = tixi3.Tixi()` cannot work. You probably meant to use the `tixi3wrapper`, see [this example](https://github.com/DLR-SC/tixi/blob/master/examples/python/tixiDemo.py).
